### PR TITLE
Bump cna to 0.12.0, enable OVS and NMState

### PR DESCRIPTION
**Pull request by phoracek**
This PR bumps CNA version to 0.12.0 https://github.com/kubevirt/cluster-network-addons-operator/releases/tag/0.12.0. It also enables NMState (node network operator) and OVS (CNI plugin).

### CODE REVIEW

**Comment by phoracek** [comment]
@rthallisey @djzager could you please take a look?

File: deploy/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-packageName: hco-operatorhub
+packageName: kubevirt-hyperconverged
 channels:
 - name: "0.0.2"
   currentCSV: kubevirt-hyperconverged-operator.v0.0.2
 - name: "0.0.1"
   currentCSV: kubevirt-hyperconverged-operator.v0.0.1
-defaultChannel: "0.0.2"
+defaultChannel: "v0.0.2"
```
Comment (rthallisey) [inline deploy/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml]: `"0.0.2"` is correct.  The listed channels are `- name: "0.0.2"` and `- name: "0.0.1"`

File: deploy/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
-packageName: hco-operatorhub
+packageName: kubevirt-hyperconverged
```
Comment (rthallisey) [inline deploy/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml]: 👍 I think the original change got picked up by accident.

File: deploy/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml (hunk 1 above)
Comment (phoracek) [reply deploy/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml]: For the record, I haven't touched this file, it was updated by `hack/build-manifests.sh`. Will manually revert the change.

**Comment by phoracek** [comment]
@rthallisey fixed the `releaseChannel`, could you please take a look?

File: deploy/olm-catalog/kubevirt-hyperconverged/0.0.2/metrics-aggregation.crd.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (SchSeba) [inline deploy/olm-catalog/kubevirt-hyperconverged/0.0.2/metrics-aggregation.crd.yaml]: nit: remove this empty line?

Comment (phoracek) [reply deploy/olm-catalog/kubevirt-hyperconverged/0.0.2/metrics-aggregation.crd.yaml]: This PR is not for CSV templates fixing, I'd rather keep this PR to the point, if that's fine

**Pull request merged**